### PR TITLE
SCMOD-3478: Update database to have global and tenant config tables

### PR DESCRIPTION
**Pull request by mmaBel**
https://jira.autonomy.com/browse/SCMOD-3478

### CODE REVIEW

**Comment by mmaBel** [comment]
This can be merged at any time. It is required by https://github.com/CAFDataProcessing/data-processing-service/pull/34.

File: Database/src/main/resources/changelog1.xml [hunk 1]
```diff
@@ -811,4 +811,31 @@
...
 
+    <changeSet author="CAF" id="1520597520212-1">
+        <createTable tableName="tbl_global_config">
+            <column name="key" type="VARCHAR(255)">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="default" type="CLOB">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="CLOB"/>
```
Comment (dermot-hardy) [inline Database/src/main/resources/changelog1.xml:822]: Let's make the description mandatory.

File: Database/src/main/resources/changelog1.xml [hunk 2]
```diff
@@ -811,4 +811,31 @@
         </insert>
     </changeSet>
 
+    <changeSet author="CAF" id="1520597520212-1">
+        <createTable tableName="tbl_global_config">
```
Comment (dermot-hardy) [inline Database/src/main/resources/changelog1.xml:815]: Can we discuss the name of this table?  I think it should be something like `tbl_tenant_settings`.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: Database/src/main/resources/changelog1.xml (hunk 1 above)
Comment (dermot-hardy) [reply Database/src/main/resources/changelog1.xml:822]: Withdrawing this comment as forcing the user may just provide a dummy description if they are forced.

File: Database/src/main/resources/changelog1.xml (hunk 2 above)
Comment (dermot-hardy) [reply Database/src/main/resources/changelog1.xml:815]: After discussion we've decided to keep the current name and will have a further Jira to introduce a `scope` column which will clarify that the configuration setting is per-tenant.

**Review by dermot-hardy — APPROVED** [review]

File: Database/src/main/resources/changelog1.xml (hunk 2 above)
Comment (dermot-hardy) [reply Database/src/main/resources/changelog1.xml:815]: https://jira.autonomy.com/browse/SCMOD-3700 is the Jira.

**Pull request merged**